specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ManagedEnvironments.json

```diff
@@ -898,6 +922,10 @@
               "description": "Vnet configuration for the environment",
               "$ref": "#/definitions/VnetConfiguration"
             },
+            "daprResiliencyDefaults": {
+              "description": "Dapr Resiliency Default Policies",
+              "$ref": "#/definitions/DaprResiliencyDefaults"
```

Comment:
>DaprResiliencyDefaults [](http://example.com/codeflow?start=37&length=22)

its not clear why the defaults are being published here. Can u provide an explanation for whats being achieved here?